### DISCUSSION TITLE
Weird see-through areas after comments

### DISCUSSION THREAD

**Discussion by tgrieger** [NvChad/NvChad#2178] [Q&A] [answered]
There are these weird see-through areas (they match whatever see-through level my terminal is set to) after any comments in any language after a fresh install of NvChad. They are pretty annoying and I was wondering if anyone knew how to make them go away?

This is on Windows Terminal. For reference, this is what it looks like
![image](https://github.com/NvChad/NvChad/assets/3431909/71a99dba-2c87-4a7d-b70f-a371e1cad061)

**Comment by siduck** [comment]
can you  please try other terminals and check?

**Reply by tgrieger** [reply]
They're not there in other terminals but they are there if I open other terminals IN windows terminal (aka, not just powershell).

**Reply by siduck** [reply]
i have no idea about this as i dont use windows,  please ask on my discord server

**Comment by tgrieger** [comment, marked as answer]
Removing italics removes the issue. Doesn't fix the underlying problem if there were italics but that at least removes the weird see-through areas.

**Reply by siduck** [reply]
@tgrieger do  you face this issue with lunarvim etc?

**Reply by tgrieger-sf** [reply]
I will give it a try tonight and let you know
